infrastructure/docker/celo/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM golang:1.14-alpine AS gobuild
+
+RUN apk add --update --no-cache \
+    g++ \
+	linux-headers \
+    make \
+    git
+
+ENV BIN_PATH=/usr/local/bin \
+    DATA_DIR=/data
```

Comment:
We usually mount volumes under `/mnt`, so maybe we could change it to `/mnt/data`? This will also affect the `start.sh` script.

Reply:
Renamed. I have also renamed `/init` to `/celo-init`, the new name should be more explicit about the purpose of this directory. Please see https://github.com/keep-network/keep-core/pull/2324/commits/8f4b4322fc20038e67e1a1d88c4057d34c6ccb67.